### DISCUSSION TITLE
ÖNEMLİ!!! ÖDÜLLER NASIL PAYLAŞILACAK...

### DISCUSSION THREAD

**Discussion by zukenberger** [IBAX-io/go-ibax#832] [Soru Cevap - Türkçe]
Toplamda 5.000.000 IBXC Airdrop tokeni dağıtılacak.
 Ama iki şekilde 1. sini anlatıyorum.
1 Şubat 2022' ödüller dağıtılmaya başlanacak. 1.airdropta 1.500.000 ibxc token verilecek
- En yüksek puana sahip ilk 1000 kişi 500 IBXC alacak.
- 1.000'den sonrakiler 300.000 IBXC'yi eşit olarak bölüşecek.
- Rastgele seçilecek 7 şanslı katılımcı 45.000 puanı aşan 100.000 IBXC alacak.
2.de ise 3.500.000 ibx token verilecek bu genel olarak referans kasanlar için olacak en çok üye getiren kişiden aşağıya doğru ödül skalası olacak. İbax.io daki rules kısmından bakabilirsiniz.

**Comment by Azizatli21** [comment]
Benim 24 bin puanım var bana ne kadar verirler acaba

**Comment by Yusufknk** [comment]
Airdrop iki şekilde olacak. A(görevler) B(Referanslar) bu ikisinden de toplam 5 Milyon dağıtılacak. Her iki airdroptan puanları ve ibxc alabilirsiniz.

**Reply by Shekilsheyy** [reply]
Puanlar hesaba direk gelmiyor görev yaptığımız halde bitti diyoruz bekleyin diyor normalmidir
